src/master_client.h

```diff
@@ -58,4 +58,16 @@ class MasterClient {
   static constexpr int64_t kSnFlushedInit = 0;
 };
 
+class RedisMasterClient : public MasterClient {
+ public:
+  virtual Status Connect(const std::string& address, int port) override;
```

Comment:
Is virtual needed here? See "Do not use virtual when declaring an override. Rationale: ..." in http://google.github.io/styleguide/cppguide.html#Inheritance